src/components/repl.tsx

```diff
@@ -97,7 +98,7 @@ const Repl: ReplProps = (props) => {
 
   let outputModel: editor.ITextModel;
   createEffect(() => {
-    const outputUri = Uri.parse(`file:///${props.id}/output_dont_import.tsx`);
+    const outputUri = Uri.parse(`file:///${props.id}/output_dont_import.${tabExtension}`);
```

Comment:
Hmm, I wonder if why this wasn't just .js given we are compiling to that? Maybe we had it as typescript for better hover tooltips or something?